src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -73,13 +81,31 @@ function ExpensifyCardPage({
                             interactive={false}
                             titleStyle={styles.newKansasLarge}
                         />
-                        {!_.isEmpty(physicalCard) && (
-                            <MenuItemWithTopDescription
-                                description={translate('cardPage.virtualCardNumber')}
-                                title={CardUtils.maskCard(virtualCard.lastFourPAN)}
-                                interactive={false}
-                                titleStyle={styles.walletCardNumber}
-                            />
+                        {!_.isEmpty(virtualCard) && (
+                            <>
+                                {shouldShowCardDetails ? (
+                                    <CardDetails
+                                        pan="1234123412341234"
+                                        expiration="11/02/2024"
+                                        cvv="321"
```

Comment:
looks great, but could you add a quick comment that these will be replaced in this issue: https://github.com/orgs/Expensify/projects/58?pane=issue&itemId=33286617

Reply:
Done!